### DISCUSSION TITLE
what's special about polyrhythm?

### DISCUSSION THREAD

**Discussion by felixroos** [tidalcycles/strudel#211] [Q&A] [answered]
Isn't polyrhythm just an alias of `stack`? It currently has its own implementation, but I think it could just be an alias. Those two do the same thing:

```js
polyrhythm([e3, g3], [e4, g4, b4])
stack([e3, g3], [e4, g4, b4])
```

**Comment by yaxu** [comment, marked as answer]
Yes agreed, IIRC they were different up until lists got reified for everything.

**Reply by felixroos** [reply]
ok, fixed in https://github.com/tidalcycles/strudel/pull/212
